modules/feature-upload.php

```diff
@@ -291,7 +304,7 @@
 				exit();
 			}
 			
-			$filepath = $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
+			$filepath = getcwd() . "/" . $env->storage_prefix . $pageindex->{$env->page}->uploadedfilepath;
```

Comment:
Ah, no worries! I know the feeling... :P

Hrm. Well, we can't have some environments working and others not working, notw can we? :P Yeah, I think I've heard something about canonical pathnames and imagick somewhere before. This looks ok, so I'll merge :smiley_cat: